index.html

```diff
@@ -62,6 +62,21 @@ <h1 id="philosophy">Our Philosophy</h1>
 
 <div class="c-header">
   <div class="l-wrapper">
+    <h1 id="philosophy">Resources</h1>
+    <p>We most certainly are not the subject matter experts in terms of the best approcahes to accessibility. Most of the work done by our team is based on a great deal of research and references form talented designers and developer accross the world...</p>
```

Comment:
Maybe something like:

We most certainly are not experts in terms of the best approaches to accessibility. Most of the work done by our team is based on a great deal of research and references. Below are some of our favorite resources.